### ISSUE TITLE
UX: vehicle part degradation is not communicated well

### ISSUE THREAD

**Issue by hexagonrecursion** [CleverRaven/Cataclysm-DDA#54244]
### Is your feature request related to a problem? Please describe.

#52744 introduced vehicle part degradation. The vehicle repair UI fails to communicate to the player how this new mechanic works. The same can be said about the item repair UI.

What the player can observe:
1. Some parts are not repaired fully when my character repairs them.
2. Some parts can not even be selected in the vehicle repair menu even though they are clearly damaged.
3. There are vertical bars next to vehicle part HP.

Questions that remain unanswered:
1. Q: Why can't I repair this? A: Because this vehicle part has suffered degradation. The more degraded a part it the less its maximum HP is.
2. Q: What causes degradation? A: The same things that cause regular damage.
3. Q: How do I repair degradation? A: You can't. You can only uninstall this part and install a new one.

### Solution you would like.

Allow any part below max HP to be selected in the vehicle repair menu even if its degradation prevents repairing it any further. When the player tries to repair a degraded part or item that is already as repaired as it is going to get, display a message box with the above three questions and answers.

### Describe alternatives you have considered.

Remove vehicle part degradation. #53431 is a much better approach to nerfing welding anyway.

**Comment by BaumgeistOne** [comment]
When I updated the game some time ago, I noticed these bars behind some of the vehicle parts. Had no idea what they were for. Noticed that if I tried to remove one of those on my helicopter, it would warn me that this change would make the heli non-flyable, which did not happen if I tried to remove a part without that bar, so I assumed those bars somehow indicated which part is structure-relevant, or something to that effect.

Sometime later I noticed I was not able to fully repair a vehicle part. Noticed that that bar was a bit smaller. Assumed that there was a connection, but since the UI in no way, shape or form told me anything about "degradation", I was unable find anything about this feature on the internet.

Long story short: I'd like to support the suggestion to implement a message about degradation into the vehicle UI. :)

**Comment by jarmush** [comment]
Totally agree. This feature was added to the game about a year ago, but in game UI I never found out what do these color bars stands for and why 'r'epair is inactive on damaged parts.
The only answer i've found is here, in github, when i searched "repair inactive" in issues. Before i found this issue, i was convinced that it's a bug or something.
